components/KiboOrderLineItem.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div data-testid="shipping-method">
+    <div :key="item.id" class="productRow">
+      <SfImage
+        class="sf-gallery__thumb"
+        :src="checkoutLineItemGetters.getProductImage(item)"
+        :alt="checkoutLineItemGetters.getProductName(item)"
+      />
+
+      <div class="content">
+        <div class="content__productName">
+          {{ checkoutLineItemGetters.getProductName(item) }}
+        </div>
+
+        <div
+          v-for="option in checkoutLineItemGetters.getProductOptions(item)"
+          :key="option.attributeFQN"
+          class="content__props"
+        >
+          <span class="title"> {{ option.name }}: </span> {{ option.value }} <br />
+        </div>
+
+        <div class="content__props">
+          <span class="title">{{ $t("Price") }}: </span> ${{
+            checkoutLineItemGetters.getProductPrice(item)
+          }}
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { SfImage } from "@storefront-ui/vue"
+import { checkoutLineItemGetters } from "@/lib/getters"
+
+export default {
+  name: "OrderLineItem",
+  components: {
+    SfImage,
+  },
+  props: {
+    item: {
+      type: Object,
+      default: () => {},
+    },
+  },
+  setup() {
+    return {
+      checkoutLineItemGetters,
+    }
+  },
+}
+</script>
+
+<style lang="scss" scoped>
+.productRow {
+  display: flex;
+  flex-direction: row;
```

Comment:
default direction is row only, do we need it to give explicitly?

Reply:
Done